backend/main/src/Group/Repositories/GroupRepository.ts

```diff
@@ -88,10 +88,20 @@ export class GroupRepository implements IGroupRepository {
 
 	/**
 	 * Delete a Group
+	 * When the group is deleted, all containers belonging to the group are also deleted.
 	 * @param id - target Group id
 	 */
 	async delete(id: GroupId): Promise<undefined> {
-		// TODO: delete all containers and users in the group.
+		// delete all containers belonging to the group.
+		const containers = await this.nishikiDbClient.listOfContainers(id.id);
+		await Promise.all(
+			containers.map((containerId) =>
+				this.nishikiDbClient.deleteContainer(containerId),
+			),
+		);
```

Comment:
When a group is deleted, the user and the group relation must be deleted.
https://nishiki-tech.github.io/nishiki-documents/project-document/database#user

Reply:
Thank you for pointing it out. I totally forgot. I need to delete explicitly related data, UserGroupGSI and invitationLInk

Reply:
I found the related indexes is deleted when you delete a data in a base table. So I just added a deletion of invitation link.

> When an application writes or deletes items in a table, any global secondary indexes on that table are updated asynchronously, using an eventually consistent model.

https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GSI.html#GSI.Writes

Reply:
I didn't know that information. Thank you.